homeassistant/components/fritz/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: still in async_setup_entry, needs to be moved to async_setup
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: one coverage miss in line 110
+  config-flow: done
```

Comment:
I believe this needs to be marked todo because the integration is missing `data_description` fields for all fields?

Reply:
Found when adding a validation rule in https://github.com/home-assistant/core/pull/131413

Reply:
nice catch :+1:  -> PR #131419